src/index.html

```diff
@@ -4,9 +4,314 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__cards">
```

Comment:
```suggestion
    <main class="main">
      <div class="catalog">
        <div class="catalog__cards">
```

Your main block needs to be a separate block. Move `catalog` block to child element. And set padding-bottom for `main`.